packages/dx-react-chart-bootstrap4/src/templates/axis/tick.test.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Tick } from './tick';
+
+const defaultProps = {
+  x1: 1, x2: 2, y1: 3, y2: 4,
+};
+
+describe('Tick', () => {
+  it('should render line with correct coordinates', () => {
+    const {
+      x1, x2, y1, y2,
+    } = shallow((
+      <Tick {...defaultProps} />
+    )).find('line').props();
+
+    expect(x1).toBe(1);
+    expect(x2).toBe(2);
+    expect(y1).toBe(3);
+    expect(y2).toBe(4);
```

Comment:
I think this way better for all tests
`.toBe(4) -> .toBe(defaultProps.y2)`

Reply:
I want to avoid all logic in a unit test. I think to compare a result with a number stronger than with object property